src/other_functions.jl

```diff
@@ -6,30 +6,23 @@
 # MIT Expat license
 #
 
-## real, imag, conj and ctranspose ##
-for f in (:real, :imag, :conj)
-    @eval ($f){T<:Number}(a::Taylor1{T}) = Taylor1(($f)(a.coeffs), a.order)
-end
-ctranspose{T<:Number}(a::Taylor1{T}) = conj(a)
-
+for T in (:Taylor1, :HomogeneousPolynomial, :TaylorN)
 
-## real, imag, conj and ctranspose ##
-for TT in (:HomogeneousPolynomial, :TaylorN), f in (:real, :imag, :conj)
-    @eval ($f){T<:Number}(a::($TT){T}) = ($TT)(($f)(a.coeffs), a.order)
-end
+    ## real, imag, conj and ctranspose ##
+    for f in (:real, :imag, :conj)
+        fdot = Symbol(f, ".")
```

Comment:
Good catch! Indeed, that was somehow problematic, and at the end $f.(...) worked fine. I'll erase it.